src/components/NumericRangeSlider.tsx

```diff
@@ -273,58 +361,40 @@ const NumericRangeSlider: React.FC<Props> = ({
                 aria-valuemax={maxBound}
                 aria-valuenow={max}
                 onMouseDown={dragHandle("max")}
-                className={`absolute -top-1 h-4 w-4 rounded-full border ${
-                  isActive ? "border-blue-700" : "border-gray-500"
-                } bg-white`}
+                className={`absolute -top-1 h-4 w-4 rounded-full border bg-white transition-shadow duration-150 ${
+                  isActive
+                    ? "border-rmiblue-800 shadow-md ring-2 ring-rmiblue-100"
+                    : "border-rmigray-400 shadow-sm"
+                } z-50`}
                 style={{ left: `calc(${maxPctOnBar}% - ${HANDLE_R}px)` }}
               />
             )}
 
-            {/* tick labels */}
+            {/* tick labels every 0.5° with °C appended */}
             {ticks.map((t) => {
               const pct = toPct(t, minBar, maxBar);
               return (
                 <div
-                  key={t}
-                  className="absolute -bottom-5 -translate-x-1/2 text-[10px] text-gray-500"
+                  key={String(t)}
+                  className="absolute -bottom-5 -translate-x-1/2 text-[12px] text-rmigray-500 z-50"
                   style={{ left: `${pct}%` }}
                 >
-                  {t}
+                  {t.toFixed(1)}°C
                 </div>
               );
             })}
           </div>
         </div>
 
-        {/* Inputs + Include-absent (checkbox to the right) */}
-        <div className="flex items-center gap-3">
-          <input
-            type="number"
-            step={step}
-            value={min ?? ""}
-            onChange={onMinInput}
-            className="w-28 rounded border px-2 py-1"
-            placeholder={`${minBound}`}
-            aria-label={`${label} min`}
-          />
-          <span className="text-gray-500">to</span>
-          <input
-            type="number"
-            step={step}
-            value={max ?? ""}
-            onChange={onMaxInput}
-            className="w-28 rounded border px-2 py-1"
-            placeholder={`${maxBound}`}
-            aria-label={`${label} max`}
-          />
-          <label className="ml-auto flex items-center gap-2 text-sm">
+        <div className="flex items-center gap-3 mt-4">
+          <div className="ml-auto flex items-center gap-2 text-sm">
```

Comment:
I think we should keep the numeric inputs. It's not immediately obvious that precise control is available in the dropdown.